Makefile

```
@@ -280,7 +283,10 @@ endif
 tools/seed7-fopen-controlled.elc: seed7-mode.elc tools/seed7-fopen-time.elc
 tools/seed7-fopen-time.elc:       seed7-mode.elc
 tools/seed7-indent-bench.elc:     seed7-mode.elc tools/seed7-fopen-time.elc
+tools/sd7-indent-perf.elc:        seed7-mode.elc
 tools/seed7-mode-time.elc:        seed7-mode.elc
```

checkmake (0.3.2)

[warning] Target "tools/seed7-mode-time.elc" defined multiple times (lines 255 and 287).

(uniquetargets)